### DISCUSSION TITLE
Can't kekulize mol

### DISCUSSION THREAD

**Discussion by andresilvapimentel** [rdkit/rdkit#5954] [Q&A]
I am trying to draw the fingerprints in a very large set of structures. So, there are many structures giving the error "Can't kekulize mol.  Unkekulized atoms" in the following code:

prints = [(m, x, bi) for x in list(result)]
figure = Draw.DrawMorganBits(prints, molsPerRow = 4, legends = [str(x) for x in list(result)])

Is there a way to work around this issue jumping it in a loop when the issue is reached?

**Comment by greglandrum** [comment]
There's a FAQ entry about kekulization failures: https://github.com/rdkit/rdkit/wiki/FrequentlyAskedQuestions#cant-kekulize-mol
